src/utils/test-utils/queries.ts

```diff
@@ -124,6 +127,7 @@ export function getMockSnap({
   },
   summary = 'Summary',
   latestVersion = '1.0.0',
+  latestChecksum = 'foo',
```

Comment:
there is a constant from test-utils, please use SNAP_SHASUM_1

Reply:
applied